hexrd/matrixutil.py

```diff
@@ -742,39 +729,15 @@ def _findduplicatevectors(vec, tol, equivPM):
 
     return eqv
 
-def normvec(v):
-    mag = np.linalg.norm(v)
-    return mag
 
-
-def normvec3(v):
-    """
-    ??? deprecated
-    """
-    mag = np.sqrt(v[0]*v[0] + v[1]*v[1] + v[2]*v[2])
-    return mag
+normvec = normvec3 = np.linalg.norm
+cross = np.cross
+determinant3 = np.linalg.det
+trace3 = np.trace
 
 
 def normalized(v):
```

Comment:
This is a silly thing to have as a function

Reply:
I agree, I just don't want to remove any functionality in case people are importing this.